frontend/packages/dev-console/src/components/projects/ProjectListPage.tsx

```diff
@@ -10,17 +10,23 @@ export interface ProjectListPageProps {
   children?: React.ReactNode;
   badge?: React.ReactNode;
 }
-const ProjectListPage: React.FC<ProjectListPageProps> = (props) => (
+const ProjectListPage: React.FC<ProjectListPageProps> = ({
+  badge,
+  title,
+  children,
+  listComponent,
+  ...listPageProps
+}) => (
   <div className="odc-project-list-page">
-    <PageHeading title={props.title} badge={props.badge}>
-      {props.children}
+    <PageHeading title={title} badge={badge}>
+      {children}
     </PageHeading>
     <hr className="odc-project-list-page__section-border" />
     <ListPage
-      {...props}
+      {...listPageProps}
```

Comment:
This avoids the whole problem by simply not passing the `badge` prop through to the `ProjectList` when it shouldn't be displayed.

Reply:
Ah, the beauty of React. Nice solution @spadgett - I'll do a quick once over in a few moments after my current review but this looks like an elegant solution.